astronomy/solar_system_dynamics_test.cpp

```diff
@@ -272,75 +275,240 @@ TEST_F(SolarSystemDynamicsTest, DISABLED_TenYearsFromJ2000) {
           /*step=*/45 * Minute));
   ephemeris->Prolong(ten_years_later.epoch());
 
+  // NOTE(phl):
+  // For Mercury and Venus the separation is about the order of magnitude we'd
+  // expect from GR for either of those bodies; since it's a combination of
+  // perihelion precession and change in anomaly, it's hard to get an exact
+  // figure for that.
+  // For Mercury the perihelion drift is what we expect from GR to within 1%.
+  // For Eris the perihelion drift is surprisingly large.  It's not clear what
+  // ephemeris HORIZONS uses either.  The inclination drift looks like a non-
+  // integrated ephemeris.
+  // For Pluto, WTF is wrong?
+  std::map<int, OrbitError> const expected_planet_orbit_errors{
+      {SolarSystemFactory::Jupiter,
+       {/*separation_per_orbit=*/0.033516 * ArcSecond,
+        /*inclination_drift_per_orbit=*/0.000014 * ArcSecond,
+        /*longitude_of_ascending_node_drift_per_orbit=*/0.010286 * ArcSecond,
+        /*argument_of_periapsis_drift_per_orbit=*/std::nullopt}},
```

Comment:
Make `argument_of_periapsis_drift_per_orbit` unconditional (but you might want to use `longitude_of_periapsis_drift_per_orbit` instead).
Same for `longitude_of_ascending_node_drift_per_orbit`.

They can be a bit ill-conditioned, but since the point is to have a change detector, let's look at them.

Reply:
Done.  As discussed, sticking to `argument_of_periapsis_drift_per_orbit`.